README.md

```diff
@@ -30,7 +30,7 @@ $ pip install matrix-nio
 
 Note that this installs nio without end-to-end encryption support. For e2ee
 support, python-olm is needed which requires the
-[libolm](https://gitlab.matrix.org/matrix-org/olm) C library.
+[libolm](https://gitlab.matrix.org/matrix-org/olm) C library (from version 3.x).
```

Comment:
It would be better to just say "(version 3.x)", else it implies we'd also support a 4.x as soon as it's released in the future, and that might not be compatible.